Application/session_manager.py

```diff
@@ -0,0 +1,173 @@
+from PySide6.QtCore import Slot, QSettings
+from PySide6.QtGui import QCloseEvent
+
+from Models.session_entity import SessionEntity
+
+
+class SessionManager:
+    """
+    SessionManager is a manager for every session that takes in data
+    sent from the controller from the view and puts it in QSettings
+    and our session entity.
+    """
+    def __init__(self):
+        """
+        Constructor - loads our session entity
+        """
+        self.session_entity = SessionEntity()
+
+    def set_session_id(self, session_id):
```

Comment:
Let's rearrange the placement of the methods to respect alphabetical ordering. This will make the file consistent with the rest of the code base.